packages/infolists/resources/views/components/entry-wrapper/index.blade.php

```diff
@@ -108,6 +108,7 @@
                     }"
                 @endif
                 @class([
+                    'flex',
```

Comment:
What's this `flex` needed for? 🙂

Reply:
weird, in my initial PR it was required to get the text to align, will remove